Kernel/Graphics/VirtIOGPU/Protocol.h

```diff
@@ -171,4 +266,68 @@ struct GetEDIDResponse {
     u8 edid[1024];
 };
 
+// No equivalent in specification
+struct ContextCreate {
+    ControlHeader header;
+    u32 name_length;
+    u32 padding;
+    AK::Array<char, 64> debug_name;
+};
+
+// No equivalent in specification
+struct ContextAttachResource {
+    ControlHeader header;
+    u32 resource_id;
+    u32 padding;
+};
+
+// No equivalent in specification
+struct CommandSubmit {
+    ControlHeader header;
+    u32 size;
+    u32 padding;
+};
+
+static_assert(sizeof(ContextCreate::debug_name) == 64);
```

Comment:
This seems like a pretty misplaced assertion, maybe put it right after the struct in question?

Reply:
Will fix in next revision.